### ISSUE TITLE
dat create not working

### ISSUE THREAD

**Issue by ralphtheninja** [datproject/dat#800]
I'm playing around with the tutorial in the documentation and I can't get `dat create` to work. What happens is the following:

1. `dat create`
2. type in title + hit enter
3. typing in the description doesn't echo something back to the terminal and hitting enter does nothing
4. Ctrl+C doesn't work either

I'm on linux x64 bit

![dat-create-bug](https://user-images.githubusercontent.com/308049/26978297-538958be-4d2b-11e7-9b4a-16a624740947.png)

**Comment by mafintosh** [comment]
Can reproduce

**Comment by ralphtheninja** [comment]
It's somewhere in here:

```js
      var schema = {
        properties: {
          title: {
            description: chalk.magenta('Title'),
            default: data.title || '',
            // pattern: /^[a-zA-Z\s\-]+$/,
            // message: 'Name must be only letters, spaces, or dashes',
            required: false
          },
          description: {
            description: chalk.magenta('Description'),
            default: data.description || ''
          }
        }
      }

      prompt.override = { title: opts.title, description: opts.description }
      prompt.message = '' // chalk.green('> ')
      // prompt.delimiter = ''
      prompt.start()
      prompt.get(schema, writeDatJson)

      function writeDatJson (err, results) {
        if (err) return exitErr(err) // prompt error
        if (!results.title && !results.description) return done()
        datjson.create(results, done)
      }
```

I can take a look and try fixing it if you like

**Comment by mafintosh** [comment]
If I downgrade to Node.js 6 it works for me. I was on 8.1.0 before

**Comment by mafintosh** [comment]
Confirmed that this is a Node 8 issue. Sources tell me a fix is coming (npm init is broken as well).
This is the cause of the bug, https://github.com/nodejs/node/pull/13560

**Comment by ralphtheninja** [comment]
Check. Have you looked anything at that prompt code? Really hairy :/

**Comment by mafintosh** [comment]
Hehe. Just waiting for a `nanoprompt` module from @yoshuawuyts

**Comment by ralphtheninja** [comment]
Anyway, maybe we can keep this issue open until a fix comes out in node 8 and I can test and verify.

**Comment by ralphtheninja** [comment]
https://gist.github.com/ralphtheninja/eb0f1f12e47dc48c170d3627fe2108d5

**Comment by mafintosh** [comment]
Fixed in node 8.1.1
